students/MicahBraun/Lesson 04/mailroom_2.py

```diff
@@ -0,0 +1,287 @@
+# ------------------------------------------------------------------------
+# NAME: MICAH BRAUN
+# PROJECT: mailroom.py
+# PURPOSE: Working with arrays(nested lists), manipulating data
+# DATE: 06/06/2018
+#
+# DESCRIPTION: Program is menu-driven user interface for interacting
+#              with a nested list of stored information. User is able
+#              to add names to a list (not currently saved upon program
+#              exit), append values to fields, check against current
+#              fields, create mock "Thank You" e-mails based on user-
+#              -input and associated fields, and generate tabular display
+#              of data for viewing.
+# ------------------------------------------------------------------------
+import time  # time to supply today's date for letters
+import os  # for default working directory
+import locale  # for currency formatting
+
+# Data -------------------------------------------------------------------
+locale.setlocale(locale.LC_ALL, '')
+today = time.strftime("%m/%d/%Y")
+
+spacing = '- ' * 56  # formatting for DONOR header
```

Comment:
For readability, it's a good idea to define your variable close to where you're using them in the code